extension/package-lock.json

```diff
@@ -1,8 +1,2890 @@
 {
     "name": "nab-al-tools",
-    "version": "0.3.39-preview.6",
-    "lockfileVersion": 1,
+    "version": "1.0.0-preview.1",
+    "lockfileVersion": 2,
```

Comment:
I guess that you're running a newer version of npm than me?
See https://stackoverflow.com/questions/64813775/is-there-any-way-to-fix-package-lock-json-lockfileversion-so-npm-uses-a-specific

Can you configure your setup to use lockfileVersion 1, or do I need to upgrade?

Reply:
I'll restore the weird parts of  package-lock.json for the time being and we'll look into upgrading npm, which I guess we should.  This was a very unintentional side effect of upgrading on my end.